src/Card/Card.tsx

```diff
@@ -70,82 +71,87 @@ const Card: React.FC<CardProps> = (props) => {
       <h3 className="card-low">
        <b>{aqi}</b>
       </h3>
-     );
+     )
     } else if (aqi >= 51 && aqi <= 100) {
      return (
       <h3 className="card-moderate">
        <b>{aqi}</b>
       </h3>
-     );
+     )
     } else if (aqi >= 151 && aqi <= 200) {
      return (
       <h3 className="card-high">
        <b>{aqi}</b>
       </h3>
-     );
+     )
     } else if (aqi >= 201 && aqi <= 300) {
      return (
       <h3 className="card-very-high">
        <b>{aqi}</b>
       </h3>
-     );
+     )
     } else if (aqi >= 301) {
      return (
       <h3 className="card-extreme">
        <b>{aqi}</b>
       </h3>
-     );
+     )
     } 
     else {
       return (<h3 className="card-extreme">
        <b>N/A</b>
-      </h3>);
+      </h3>)
     }
   }
 
   const temp = (temp: number) => {
    if (temp <= 32) {
     return (
      <h3 className="card-extreme"><b>{temp}</b></h3>
-    );
+    )
    } else if (temp >= 33 && temp <= 59) {
     return (
      <h3 className="card-moderate"><b>{temp}</b></h3>
-    );
+    )
    } else if (temp >= 60 && temp <= 80) {
     return (
      <h3 className="card-high"><b>{temp}</b></h3>
-    );
+    )
    } else if (temp >= 80) {
     return (
      <h3 className="card-very-high"><b>{temp}</b></h3>
-    );
+    )
    }
-  };
+  }
 
   const weatherIcon = (icon: string) => {
-    if (icon === '01d') {
-      return (<img src={clearDay} alt="Sun shining" />);
+    if (icon === "01d") {
```

Comment:
We may be able to make these functions module exports I may be able to utilize in Location. 
Great Work, looks awesome.